command/run/run.go

```diff
@@ -315,6 +315,8 @@ func Command() *cobra.Command {
 		apiServerConfig:     forwarder.DefaultHTTPServerConfig(),
 		logConfig:           log.DefaultConfig(),
 	}
+	c.httpTransportConfig.PromRegistry = c.promReg
+	c.httpProxyConfig.PromNamespace = promNs
```

Comment:
Should `c.httpProxyConfig.PromNamespace = promNs` be httpTransportConfig? I see c.httpProxyConfig.PromNamespace being set twice here.

Reply:
Correct.